### PR TITLE
Fix model patience logging

### LINKED ISSUE #151: bug in Model.fit method -- break should be indented, causes training to stop early
in code block below, currently the training breaks out of the loop iterating through epochs whenever validation accuracy has **not** improved, because the `break` statement is not indented

https://github.com/NickleDave/vak/blob/e292610a2065933cfe6a8ba088c4333561226e88/src/vak/engine/model.py#L301

```
                        if patience:
                            if epoch_acc > max_val_acc:
                                max_val_acc = metric_vals['acc']
                                patience_counter = 0
                                log_or_print('accuracy improved, saving checkpoint', logger=self.logger, level='info')
                                self.save(ckpt_path, epoch)
                            else:
                                patience_counter += 1
                                if patience_counter > patience:
                                    log_or_print(
                                        f'early stopping, validation accuracy has not improved in {patience} epochs',
                                        logger=self.logger, level='info')
                                    if not save_best_only:
                                        self.save(ckpt_path, epoch)
                                break
```

fix should
- [ ] indent this
- [ ] add logging statement for when validation accuracy has not improved but we are not stopping yet (because `patience_counter < patience`)

### PR DESCRIPTION
**Pull request by NickleDave**
@yardencsGitHub this should fix #151 -- training a model to test the fix now

### CODE REVIEW

**Comment by NickleDave** [comment]
Update: seems to have fixed it
